docs/03-design/LG-03-07.adoc

```diff
@@ -2,22 +2,38 @@
 // tag::DE[]
 
 [[LG-03-07]]
-==== LZ 03-07 [ehemaliges LZ 2-09]: Schnittstellen entwerfen und festlegen (R1-R3)
+==== LZ 03-07 [ehemaliges LZ 2-09]: Schnittstellen entwerfen und spezifizieren (R1-R3)
 
-Softwarearchitekt:innen kennen die hohe Bedeutung von Schnittstellen. Sie können Schnittstellen zwischen Architekturbausteinen sowie externe Schnittstellen zwischen dem System und Elementen außerhalb des Systems entwerfen bzw. festlegen.
+Softwarearchitekt:innen kennen die kritische Bedeutung von Schnittstellen für die Interaktion zwischen Architekturbausteinen oder zwischen dem System und externen Elementen. 
+Sie können solche Schnittstellen entwerfen und spezifizieren.
 
 Sie kennen:
 
-* wünschenswerte Eigenschaften von Schnittstellen und können diese bei ihrer Entwicklung einsetzen:
+* wünschenswerte Eigenschaften von Schnittstellen und können diese beim Entwurf erreichen (R1):
 ** einfach zu erlernen, einfach zu benutzen, einfach zu erweitern
 ** schwer zu missbrauchen
 ** funktional vollständig aus Sicht der Nutzer:innen oder nutzender Bausteine.
-* die Notwendigkeit unterschiedlicher Behandlung interner und externer Schnittstellen
-* die Trennung zwischen Schnittstelle und Implementierung:
+
+* die Notwendigkeit unterschiedlicher Behandlung interner und externer Schnittstellen (R2)
+
+* die Trennung zwischen Schnittstelle und Implementierung (R1):
 ** Implementierungen können bei Bedarf ausgetauscht werden.
-* unterschiedliche Implementierungsansätze von Schnittstellen (R3):
-** ressourcenorientierter Ansatz (REST, REpresentational State Transfer)
-** serviceorientierter Ansatz (wie bei WS-*/SOAP-basierten Webservices).
+
+* unterschiedliche Charakteristika von Schnittstellen, beispielsweise (R3):
+** Transportkanäle (etwa: das OSI-7-Schichtenmodell oder TCP/IP)
```

Comment:
Some changes have gone missing here (see English translation)

Reply:
fixed that sentence + two typos (upper/lowercase)